en/flight_controller/kakuteh7mini.md

```diff
@@ -0,0 +1,143 @@
+# Holybro Kakute H7 mini
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+Contact the [manufacturer](https://shop.holybro.com/) for hardware support or compliance issues.
+:::
+
+The [Holybro Kakute H7 mini](http://www.holybro.com/product/kakute-h7-mini/) flight controller is intended for lightweight frame builds (such as racers, etc.).
+
+ This flight controller is full of features including integrated Bluetooth, HD camera plug, dual plug-and-play 4in1 ESC ports, 9V VTX ON/OFF Pit Switch, barometer, OSD, 6x UARTs, 128MB Flash for logging (not supported with PX4 yet), 5V BEC, and bigger soldering pad with easy layout and much more.
+
+The Kakute H7 mini builds upon the best features of its predecessor, the [Kakute F7](../flight_controller/kakutef7.md), and the [Kakute H7](../flight_controller/kakuteh7.md).
+
+The board also has an on-board barometer, LED & buzzer pad, and I2C pad (SDA & SCL) for external GPS/magnetometers.
+
+<img src="../../assets/flight_controller/kakuteh7mini/kakuteh7mini_top.jpg" width="300px" title="KakuteH7Mini Top Image" /> <img src="../../assets/flight_controller/kakuteh7mini/kakuteh7mini_bottom.jpg" width="300px" title="KakuteH7Mini Bottom Image" />
+
+:::note
+This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+:::
+
+:::note
+PX4 runs on the H7 mini v1.3 and later.
+:::
+
+## Key Features
+
+- MCU: STM32H743 32-bit processor running at 480 MHz
+- IMU: BMI270
+- Barometer: BMP280
+- OSD: AT7456E
+- VTX On/Off Pit Switch: Not used with PX4
+- 6x UARTs (1,2,3,4,6,7; UART2 is used for Bluetooth telemetry)
```

Comment:
@vincentpoont2 In your comment in https://github.com/PX4/PX4-user_guide/pull/2247#discussion_r1099020115 , I think you are saying that bluetooth and the Pit switch are not on the mini - right? So this change is needed.

```suggestion
- OSD: AT7456E
- 6x UARTs (1,2,3,4,6,7)
```

Reply:
Sorry, it has the VTX pit switch, but no bluetooth. I have made the edit.